lib/Utils/TwigExtensionFlynt.php

```diff
@@ -25,11 +26,21 @@ public function getFunctions()
 
     public function renderFlexibleContent(Twig_Environment $env, $context, $fields, $withContext = true, $ignoreMissing = false, $sandboxed = false)
     {
-        $output = '';
-        foreach ((empty($fields) ? [] : $fields) as $field) {
-            $output .= $this->renderComponent($env, $context, ucfirst($field['acf_fc_layout']), $field, $withContext, $ignoreMissing, $sandboxed);
+        if (!Defaults::$useGutenberg) {
```

Comment:
Any thoughts on this one? As you can see I rely on `the_content` for Gutenberg.

Has some benefits f.e. RSS support, post excerpts, etc. but won't allow you to have multiple flexibleContent renderers on one page.

Any thoughts @domtra?